src/Backend/Core/Entity/Module.php

```diff
@@ -0,0 +1,106 @@
+<?php
```

Comment:
I think this entity should be in the Extensions module, and not in the groups module.